tests/test.py

```diff
@@ -690,6 +642,18 @@ def setUpModule():
     drop_tables()
 
 
+def extract_client_id_from_cert(tls_cert, extractor='distinguished_name'):
```

Comment:
because of found several usage of these constant value `distinguished_name`, can we create some global value for it? Or constant class value like:
```
class TLSClientIDSource:
  DN = 'distinguished_name'
  SERIAL = 'serial_number'
```
and use it instead of hardcoded values?

Reply:
Lets create corresponded global variables